sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -612,19 +645,19 @@ def hpat_pandas_series_nsmallest(self, n=5, keep='first'):
     Returns
     -------
     :obj:`series`
-         returns :obj:`series`
+        returns :obj:`series`
```

Comment:
I think that with new docs generation system it is not necessary to copy parameters and returns description from Pandas docs.
@samaid do you agree?

Reply:
@PokhodenkoSA so what the decision?

Reply:
@samaid agree to remove it.